.github/workflows/i18n-validate-builds-submodule.yml

```diff
@@ -0,0 +1,42 @@
+name: i18n - Build Validation - Submodule
```

Comment:
This one should go in the submodule repo, tbh. That's where we'd want to validate things.

Reply:
Does it have to though? The PR to update the submodule will come in and this will run - so we will know if it fails.

Reply:
I think we can move forward with this approach and maybe move this to the i18n repo later.

Reply:
Actually, I think it _can't_ work if it runs on the submodule.

The problem is the client is built from the main repo (using a specific commit of the i18n repo). So, if someone updates the 18n repo, audit passes and then we merge a breaking change to the English, we can't build.